src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup/help/Backup-AzureRmRecoveryServicesBackupItem.md

```diff
@@ -42,6 +44,19 @@ The last command triggers the backup job for the Backup item in $Item.
 
 ## PARAMETERS
 
+### -DefaultProfile
+The credentials, account, tenant, and subscription used for communication with azure.```yaml
```

Comment:
@dragonfly91 for all of the files where you find this applies, please make sure the ` ```yaml ` string at the end of this parameter description is moved to a line by itself. See other parameters in this file for how this should look.